tests/templates/kuttl/mount-dags-gitsync/dags/pyspark_pi.py

```diff
@@ -0,0 +1,57 @@
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.exceptions import AirflowException
+from airflow.utils import yaml
+import os
+import sys
+print(sys.path)
+
+from stackable.spark_kubernetes_sensor import SparkKubernetesSensor  # noqa: E402
```

Comment:
Have you actually ever tried relative imports (starting with a dot) ?

```suggestion
from .stackable.spark_kubernetes_sensor import SparkKubernetesSensor  # noqa: E402
```

They should not require changes to PYTHONPATH since they are ... well ... relative to this script.